### ISSUE TITLE
ENHANCEMENT REQUEST - Hashcat running output 

### ISSUE THREAD

**Issue by MidnightSeer** [ctxis/crackerjack#2]
Hello,

First off, love your tool; it offers a perfect mix of simplicity, security, and job control for hashcat over a web interface!  I would like to make a suggestion: can you add a dropdown section within running sessions that displays the current hashcat output?  The hashcat output provides % and time estimates that are very helpful in addition to the current increment count (for bruteforce).  Hashcat also refreshed this screen if you press any key other than those mentioned on the screen.

Either way, still a great tool.  Thank you for taking your time to create this project and make it open source!

UPDATE:
Apparently the session timed out...when I restored the screen it showed an integer overflow error...maybe I messed up the masked?  All i wanted was a simple bruteforce mode attack so i manually (in the screen session) entered the -a -i --increment-min 10 --increment-max 10 -w 4 and the online session refreshed (which was cool!).

**Comment by sadreck** [comment]
Hi and thank you for the great feedback!

If I'm not mistaken what you're asking already exists if you click the "View Raw/Debug Information" button:

![image](https://user-images.githubusercontent.com/3439227/85106717-91907c00-b204-11ea-83f9-d931f6adf872.png)

With regards to your update, the web gui already supports the `--increment-min` and `--increment-max` parameters, unless you were talking about adding `-w`? Also yes - as long as you use the same session hashcat name the GUI will pick it up regardless where it was started from. You can get the session name either from the "Raw Information" screenshot above (first line, "Session"), or by hovering your mouse over the session title:

![image](https://user-images.githubusercontent.com/3439227/85107065-3743eb00-b205-11ea-8787-7c36952074ee.png)

Thanks,
Pavel

**Comment by MidnightSeer** [comment]
yes this is great!  My impatient self forgot all the procedures hashcat runs through before it starts cracking so I thought it was broken before it even started.  Also i normally run hashcat without specifying a mask, so this tripped me up a little bit but I eventually figured it out.  

On the point about the -w option, this would be a great addition.  I have found myself starting and stopping hashcat processes only to recover the screen and add the -w (3|4) to the command.  Thanks for the responsiveness and again, great tool!
